scripts/aws/templates/index.js.j2

```diff
@@ -178,14 +179,23 @@ exports.handler = function(event, context) {
     logger.log('value2 =', event.key2);
     logger.log('value3 =', event.key3);
     
-    //log JSON objects
-    logger.log(event);
-    
-    //specify the timestamp explicitly, useful for forwarding events like from AWS IOT
-    logger.logWithTime(Date.now(), event);
+    var decode = new Buffer(event.awslogs.data, 'base64');
+    zlib.gunzip(decode, function(e, result) {
+        if (e) {
+            context.fail(e);
+        } else {
+            result = JSON.parse(result.toString('ascii'));
+            console.log("Event Data:", JSON.stringify(result, null, 2));
```

Comment:
I don't think we want to log unconditionally to the console, I would remove this.